packages/pwa-kit-react-sdk/src/webpack/config.js

```diff
@@ -189,13 +190,15 @@ const common = {
         new CopyPlugin({
             patterns: [
                 {from: 'app/static/', to: 'static/'},
-                {
-                    from: 'config/',
-                    to: 'config/',
-                    globOptions: {
-                        ignore: ['**/local.*']
+                ...[
```

Comment:
Is the [`noErrorOnMissing`](https://www.npmjs.com/package/copy-webpack-plugin#noerroronmissing=) option applicable here? Seems like that would be the cleanest solution.

If that doesn't work, you can still simplify a bit:
```ts
            patterns: [
                {from: 'app/static/', to: 'static/'},
                fs.existsSync(configDir) && {
                    from: 'config/',
                    to: 'config/',
                    globOptions: {
                        ignore: ['**/local.*']
                    }
                }
            ].filter(Boolean)

Reply:
I've tried that option, but does not work in V1. While adding [noErrorOnMissing](https://github.com/webpack-contrib/copy-webpack-plugin#noerroronmissing) does suppress the error message, it does not prevent the server from continuously restarting.. see the PR description for a video.

Reply:
My bad for missing that! 😓

Reply:
The issue with the restarting is the existence of the "config" folder still being in the build directory. If you were to clean your project up `git clean -xdf` you'll see that that doesn't happen anymore. 

I wonder if we need to blow away that folder so we don't run into this issue if someone was to say, move the config from the config folder into the package.json without cleanup the build folder. 


this below works for me. 

https://github.com/SalesforceCommerceCloud/pwa-kit/compare/copy-config-dir...copy-config-dir-2